wiki/Featured_Artists/en.md

```diff
@@ -0,0 +1,13 @@
+# Featured Artists
+
+<!-- TODO: include some history about featured artists / mappers' guild, maybe in /History_of_osu! too -->
+
+**Featured artists** are music artists that have worked with osu! to make a selection of their tracks available for unrestricted use in the community. Some of the featured artists have also composed tracks specifically for osu! and released them as osu! exclusives.
```

Comment:
unrestricted use -> fair use?

Reply:
i dont think that's what fair use means (if you're talking about copyright-related stuff)

Reply:
yeah "fair use" implies that you're working around a copyright, but in this case osu! has license to use the music so they don't have to worry about fair use